ophirofox/content_scripts/lobs.css

```
@@ -0,0 +1,24 @@
+.ophirofox-europresse {
+	background-color: #fed500;
+    padding: 1rem 1.5rem;
+    color: #000;
```

ici il y a un pb d'identation 

```suggestion
    background-color: #fed500;
    padding: 1rem 1.5rem;
    color: #000;
```